lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -453,5 +460,34 @@ namespace Js
         return obj;
     }
 
+    Var JsBuiltInEngineInterfaceExtensionObject::EntryJsBuiltIn_Internal_ArraySpeciesCreate(RecyclableObject *function, CallInfo callInfo, ...)
+    {
+        EngineInterfaceObject_CommonFunctionProlog(function, callInfo);
+        Assert(args.Info.Count == 3);
```

Comment:
>Assert [](start = 8, length = 6)

AssertOfFailFast

Reply:
I chose a normal Assert because the function is callable only through the JS built-ins and not in any user controlled way. If you think it needs to be a FailFast I'll change it.

Reply:
It should be better be failfast it is just a check.

---
In reply to: [187739964](https://github.com/Microsoft/ChakraCore/pull/5137#discussion_r187739964) [](ancestors = 187739964)